sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs

```diff
@@ -24,16 +27,21 @@ public class FormRecognizerClient
         private readonly FormRecognizerClientOptions _options;
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="FormRecognizerClient"/>.
+        /// Initializes a new instance of the <see cref="FormRecognizerClient"/> class.
         /// </summary>
+        /// <param name="endpoint">The endpoint to use for connecting to the Form Recognizer Azure Cognitive Service. The URI is likely to be similar to <c>{protocol}://{resourcename}.cognitiveservices.azure.com</c>.</param>
```

Comment:
I'd avoid listing out the URI format here ... maybe refer to the README for details, or say that the endpoint string can be found in the Azure portal?